AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoInternalConstants.h

```diff
@@ -387,3 +387,12 @@ extern NSString * const kAlfrescoPersonPropertyCompanyEmail;
 extern NSString * const kAlfrescoDefaultBackgroundNetworkSessionId;
 extern NSString * const kAlfrescoDefaultBackgroundNetworkSessionSharedContainerId;
 
+/**---------------------------------------------------------------------------------------
+ * @name Cloud Connection Status
+ --------------------------------------------------------------------------------------- */
+typedef NS_ENUM(NSInteger, CloudConnectionStatus)
```

Comment:
Ok, will add the prefix for this and for ConnectionDiagnosticStatus? Is that ok?